src/js/subtraction/components/Detail/__tests__/__snapshots__/EditSubtraction.test.tsx.snap

```diff
@@ -0,0 +1,216 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`<EditSubtraction /> > should render after name is changed 1`] = `
+<Modal
+  label="Edit Subtraction"
+  onHide={[MockFunction spy]}
+  show={true}
+  size="sm"
+>
+  <ModalHeader>
+    Edit Subtraction
+  </ModalHeader>
+  <form
+    onSubmit={[Function]}
```

Comment:
Are these being used? looks like you did an RTL conversion for `EditSubtracton.test.tsx` so guessing these are vestigial